.gitignore

```diff
@@ -0,0 +1,133 @@
+# Environmental variable file
+.env
+.env.test
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+lerna-debug.log*
+.pnpm-debug.log*
+
+# Diagnostic reports (https://nodejs.org/api/report.html)
+report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
+
+# Runtime data
```

Comment:
Theres an awful lot of files referenced in here that are not in the project, it would be good to remove all the unnecessary noise in here. A good way to do this is to install the project and use `git status` to see what files would be committed, Id expect most of these could be removed